element_optogenetics/optogenetics.py

```diff
@@ -110,8 +110,8 @@ class Ramp(dj.Part):
         definition = """
         -> master
         ---
-        ramp_up_proportion   : decimal(2, 2) # ramp up prop of the linear waveform
-        ramp_down_proportion : decimal(2, 2) # ramp down prop of the linear waveform
+        ramp_up_proportion   : decimal(2, 2) # Ramp up proportion of the linear waveform
```

Comment:
can this not be 1.00?  `decimal(2, 2)` can accommodate `0.00` to `0.99` but cannot do `1.00`. You may also want to make it `unsigned`. It's signed by default.

Reply:
Thanks @dimitri-yatsenko.  Good catch.  I agree with making those changes.

Reply:
Sure, I can add `unsigned` across these cases. I think the exclusion of `1.00` here is intentional. The signal must be ramping down for some proportion of the time, even if that's 1%